sphinx_needs/utils.py

```diff
@@ -581,3 +584,32 @@ def add_doc(env: BuildEnvironment, docname: str, category=None):
             env.needs_all_docs[category] = []
         if docname not in env.needs_all_docs[category]:
             env.needs_all_docs[category].append(docname)
+
+
+def plantuml_debug(uml_type, cmd, code, docname, lineno):
+    """Checks a given Plantuml Code and prints found errors"""
+    try:
```

Comment:
Perhaps it would be better here to simply use: https://github.com/sphinx-contrib/plantuml/blob/36716425827d26502552f23f1582069878934514/sphinxcontrib/plantuml.py#L230

Reply:
But I don't get any information back if problems have happened.